ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.statistical_dtype_values(function="dist"),
+    dtype_and_y=helpers.statistical_dtype_values(function="dist"),
+    as_variable=helpers.array_bools(num_arrays=2),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.torch.dist"
+    ),
+    native_array=helpers.array_bools(num_arrays=2),
+    p=st.integers(),
```

Comment:
please use the hypothesis strategy of 
st.sampled_from( st.integers(), None) , this will enable hypothesis to generate None values for p since it is optional.

Reply:
Okay !